CHANGELOG.md

```diff
@@ -149,7 +149,7 @@ These changes are available on the `master` branch, but have not yet been releas
 ### Removed
 
 - Removed deprecated support for `Option` in `BridgeCommand`. Use `BridgeOption`
-  instead. ([#2731])(https://github.com/Pycord-Development/pycord/pull/2731))
+  instead. ([#2731](https://github.com/Pycord-Development/pycord/pull/2731))
```

Comment:
:information_source: The fact that it's not on a new line is due to pre-commit

Reply:
annoying :/